src/geometricfigures/Square.java

```diff
@@ -0,0 +1,12 @@
+package geometricfigures;
+public record Square(double side) implements GeometricFigure {
```

Comment:
what is the profit of record in this case?

Reply:
The getter and constructor are auto-generated.